news/fix-non-callable-console-spam.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* <news item>
+
+**Deprecated:**
+
+* <news item>
+
+**Removed:**
+
+* <news item>
+
+**Fixed:**
+
+* Fixed annoying console spam when calling a non-callable
```

Comment:
can it more descriptive without such strong words. Like, `hide warnings when autocompleting`